person.rb

```diff
@@ -1,15 +1,16 @@
 require './decorators'
 
 class Person < Nameable
-  def initialize(age, name = 'unknown', parent_permission: true)
+  def initialize(age, rentals, name = 'unknown', parent_permission: true)
     @id = Random.rand(1...1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = rentals
     super()
   end
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] In your person class, there should be a method for adding rentals, just as it should be in the book class as well. Remember that the rental class receives a person and a book, so you need to set them in the correspondent classes as well.